{"name": "test_01[iozone -r 1M -i 0 -i 1  -s 20G -I]", "status": "broken", "statusDetails": {"message": "TypeError: not all arguments converted during string formatting", "trace": "self = <Case.test_fio.TestIOzone object at 0x0000018F6F051810>\ncmd = 'iozone -r 1M -i 0 -i 1  -s 20G -I'\n\n    @pytest.mark.parametrize('cmd', [i[0] for i in data])\n    def test_01(self, cmd):\n        ioz_res = fp.send(cmd)\n>       fp.assert_Iozone(ioz_res)\n\ntest_fio.py:16: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\Utils\\Base_init.py:59: in assert_Iozone\n    print(self.log.warning('断言失败', f'\\n历史数据{dict_list['history_result']}' + f'\\n测试数据：{return_res}'))\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1531: in warning\n    self._log(WARNING, msg, args, **kwargs)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1664: in _log\n    self.handle(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1680: in handle\n    self.callHandlers(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1736: in callHandlers\n    hdlr.handle(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1026: in handle\n    self.emit(record)\nC:\\Users\\yi.mei-ext\\AppData\\Roaming\\Python\\Python313\\site-packages\\_pytest\\logging.py:386: in emit\n    super().emit(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1158: in emit\n    self.handleError(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:1150: in emit\n    msg = self.format(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:998: in format\n    return fmt.format(record)\nC:\\Users\\yi.mei-ext\\AppData\\Roaming\\Python\\Python313\\site-packages\\_pytest\\logging.py:139: in format\n    return super().format(record)\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:711: in format\n    record.message = record.getMessage()\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <LogRecord: BspTest, 30, C:\\Programs\\BSPTest\\Utils\\Base_init.py, 59, \"断言失败\">\n\n    def getMessage(self):\n        \"\"\"\n        Return the message for this LogRecord.\n    \n        Return the message for this LogRecord after merging any user-supplied\n        arguments with the message.\n        \"\"\"\n        msg = str(self.msg)\n        if self.args:\n>           msg = msg % self.args\nE           TypeError: not all arguments converted during string formatting\n\nC:\\Program Files\\Python313\\Lib\\logging\\__init__.py:400: TypeError"}, "attachments": [{"name": "log", "source": "7667b460-8465-4caf-a0bf-9c333a2216c6-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "cmd", "value": "'iozone -r 1M -i 0 -i 1  -s 20G -I'"}], "start": 1740135357335, "stop": 1740135396955, "uuid": "330e4ac5-f8f9-4a26-8837-12d70def6677", "historyId": "1800ecaa154f528cbb1cc87e4fcc3d7a", "testCaseId": "01a55d7010f48ebe0c3cf2503891f546", "fullName": "Case.test_fio.TestIOzone#test_01", "labels": [{"name": "parentSuite", "value": "Case"}, {"name": "suite", "value": "test_fio"}, {"name": "subSuite", "value": "TestIOzone"}, {"name": "host", "value": "Meiyi-EXT"}, {"name": "thread", "value": "27928-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Case.test_fio"}]}